File: RepoTransBench/source_projects/Python/BBuf_onnx_learn/test_onnxapi_creat_onnx_example.py
import unittest
from onnxapi.creat_onnx_example import make_identity, sum_list

class TestOnnxApiCreatOnnxExample(unittest.TestCase):

    def test_make_identity(self):
        self.assertEqual(make_identity(100), 100)
        self.assertEqual(make_identity([1,2,3]), [1,2,3])

    def test_sum_list(self):
        self.assertEqual(sum_list([1,2,3]), 6)
        self.assertEqual(sum_list([]), 0)

if __name__ == "__main__":
    unittest.main()

File: RepoTransBench/target_projects/Python/Java/BBuf_onnx_learn/src/test/java/com/example/original/TestOnnxApiCreatOnnxExample.java
package com.example.original;

import com.example.onnxapi.CreatOnnxExample;
import org.junit.jupiter.api.Test;

import java.util.Arrays;
import java.util.Collections;

import static org.junit.jupiter.api.Assertions.*;

class TestOnnxApiCreatOnnxExample {

    @Test
    void testMakeIdentity() {
        assertEquals(100, (int)CreatOnnxExample.makeIdentity(100));
        assertEquals(Arrays.asList(1, 2, 3), CreatOnnxExample.makeIdentity(Arrays.asList(1, 2, 3)));
    }

    @Test
    void testSumList() {
        assertEquals(6, CreatOnnxExample.sumList(Arrays.asList(1, 2, 3)));
        assertEquals(0, CreatOnnxExample.sumList(Collections.emptyList()));
    }
}